src/containers/SuperAdminOrganizations.tsx

```diff
@@ -0,0 +1,180 @@
+    setDialogOpen(true);
+    setOrganizationIdToEdit(organizationId);
+    setOrganizationActiveType(active);
+  };
+
+  const closeDialog = () => {
+    setDialogOpen(false);
+    setOrganizationActiveType(null);
+    setOrganizationIdToEdit(null);
+  };
+
+  const submitEditOrganization = async () => {
+    closeDialog();
+    await editOrganizationsActiveMutation({
+      variables: {
+        organizationId: organizationIdToEdit,
+        active: organizationActiveType
+      }
+    });
+    await refetchOrganizations();
+  };
+
+  const columns: GridColDef[] = [
+    {
+      field: "id",
+      headerName: "ID",
+      flex: 1
+    },
+    {
+      field: "name",
+      headerName: "Name",
+      flex: 7
+    },
+    {
+      field: "manageOrganization",
+      headerName: "Manage",
+      sortable: false,
+      flex: 3,
+      renderCell: (params: GridRenderCellParams) => (
+        <Button
+          onClick={() => openOrganizationSettings(params.row.id)}
+          startIcon={<OpenInNewIcon />}
+          color="primary"
+          variant="contained"
+        >
+          Manage
+        </Button>
+      )
+    },
+    {
+      field: "activateDeactive",
+      headerName: "Activate / Deactivate",
+      sortable: false,
+      flex: 3,
+      renderCell: (params: GridRenderCellParams) => (
+        <div>
+          {params.row.deletedAt ? (
+            <Button
+              onClick={() => editOrganization(params.row.id, true)}
+              startIcon={<RestoreFromTrashIcon />}
+              color="primary"
+              variant="contained"
+            >
+              Reactivate
+            </Button>
+          ) : (
+            <RedButton
+              onClick={() => editOrganization(params.row.id, false)}
+              startIcon={<DeleteIcon />}
+              variant="contained"
+            >
+              Deactivate
+            </RedButton>
+          )}
+        </div>
+      )
+    }
+  ];
+
+  return loading ? (
+    <div>Loading...</div>
+  ) : (
+    <div>
+      <Paper>
+        <DataGridPro
+          disableSelectionOnClick
+          autoHeight
+          columns={columns}
+          rows={organizationsData?.organizations ?? []}
+          rowsPerPageOptions={[25]}
+          pagination
+          pageSize={25}
+        />
+      </Paper>
+      <Dialog open={dialogOpen} onClose={closeDialog}>
+        <DialogTitle>
+          {organizationActiveType
+            ? "Reactivate Organization?"
+            : "Deactivate Organization?"}
+        </DialogTitle>
+        <DialogContent>
+          <DialogContentText>
+            Are you sure you want to{" "}
+            {organizationActiveType ? "reactivate" : "deactivate"} this
+            organization?
+            {!organizationActiveType && (
+              <span>
+                <br />
+                Deactivating this organization will suspend all current users in
+                that organization.
```

Comment:
Edit to add organizing team suggestion
```suggestion
                Deactivating this organization will suspend current users in this organization: ({Name}). Users with access to other organizations will retain that access.
```